ci-operator/config/red-hat-data-services/notebooks/red-hat-data-services-notebooks-release-2024b.yaml

```diff
@@ -0,0 +1,317 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.17-openshift-4.10
+images:
+- context_dir: base/rhel9-python-3.9
+  dockerfile_path: Dockerfile
+  to: base-rhel9-python-3.9
+- context_dir: base/ubi9-python-3.11
+  dockerfile_path: Dockerfile
+  to: base-ubi9-python-3.11
```

Comment:
Hm, there is not python-3.11 based directory in https://github.com/red-hat-data-services/notebooks/tree/release-2024b/base . How was that branch created? Was it forked from main or not? :thinking:

Reply:
That's true, python-3.11 flavor is not there yet, we will fetch them once[ pull bot ](https://github.com/red-hat-data-services/notebooks/blob/main/.github/pull.yml#L17) get in action. Where it will update the [release-2024b](https://github.com/red-hat-data-services/notebooks/tree/release-2024b/base) from [2024b](https://github.com/opendatahub-io/notebooks/tree/2024b/base)